cloud_kafka/librdkafka/avro_producer.cc

```diff
@@ -0,0 +1,132 @@
+#include <stdio.h>
+
+#include <memory>
+#include <string>
+#include <utility>
```

Comment:
```suggestion
```

Reply:
It's imported for `std::pair`. https://en.cppreference.com/w/cpp/utility/pair